### ISSUE TITLE
High CPU usage

### ISSUE THREAD

**Issue by idrakimuhamad** [vincaslt/vscode-highlight-matching-tag#36]
Up till now, the plugin will use almost 100% CPU and will stay in the process even after you've disabled it or close VSCode. You'll have to manually kill the process.

For now, I have to uninstall the plugin and looking at some other alternative.

**Comment by hobbyman** [comment]
I've been having a 100% CPU, on a Mac, every now and then on large PHP files. I would quit VS Code and the "Code Helper" process would continue at 100%. Would have to kill that process. I've tried over and over to figure out what was going on. 

It happened to me again this morning. Took me over an hour to figure out it was this plugin causing the issue. Uninstalled and my issue went away.

**Comment by wulffeld** [comment]
This snippet will cause 100% cpu when edited.

```
#
module A
  module B
    module C
      module D
        module E
          class F

            description <<-EOD.strip_heredoc
            Blah blah
            EOD

          end
        end
      end
    end
  end
end
```

When the extensions is disabled it works fine.

**Comment by vincaslt** [comment]
Should be fixed with v0.5, test cases cover it now.
